var/spack/repos/builtin/packages/nalu-wind/package.py

```diff
@@ -114,16 +96,30 @@ class NaluWind(CMakePackage, CudaPackage, ROCmPackage):
     # indirect dependency needed to make original concretizer work
     depends_on("netcdf-c+parallel-netcdf")
     depends_on("boost +filesystem +iostreams cxxstd=14", when="+boost")
-    supported_cxxstd = ["17"]
-    variant(
-        "cxxstd", default="17", values=supported_cxxstd, multi=False, description="cxx standard"
+    depends_on("hypre+gpu-aware-mpi", when="+gpu-aware-mpi")
+    depends_on("hypre+umpire", when="+umpire")
+    depends_on("trilinos~shared", when="+trilinos-solvers")
```

Comment:
Is this still just `darwin` or always?

Reply:
That's a good question. Maybe @marchdf or @itopcuoglu can answer that.

Reply:
I have only experienced the issue on my mac, so I think it is darwin only.